test/unit/query_components/facets_test.rb

```diff
@@ -2,11 +2,19 @@
 require "unified_search_builder"
 
 class FacetsTest < ShouldaUnitTestCase
+  def make_search_params(facets:, filters: [])
```

Comment:
It's not absolutely necessary, but it makes things a bit simpler. Because the expectations are against the elastic search query structure in these tests, our change results in all of them changing, unless the debug flag is set, despite it not making a difference to the end search result for most content.

Even though the new behaviour is on by default, I think it's simpler and clearer to keep the old semantics for these searches, and add new tests for the new behaviour (excluding withdrawn).